src/Registration.vue

```diff
@@ -0,0 +1,162 @@
+<!--
+ - @copyright Copyright (c) 2018 Joas Schilling <coding@schilljs.com>
+ - @copyright Copyright (c) 2019 Gary Kim <gary@garykim.dev>
```

Comment:
copyright seems inaccurate given that this is a new file

Reply:
copy pasta :D